themes/red-theme/lms/templates/header.html

```diff
@@ -43,7 +43,7 @@
     <h1 class="logo">
       <a href="${marketing_link('ROOT')}">
         <%block name="navigation_logo">
-            <img src="${static.url("images/logo.png")}" alt="${platform_name()}"/>
+            <img src="${static.url("images/logo.png")}" alt="${_("{platform_name} Home Page").format(platform_name=settings.PLATFORM_NAME)}"/>
```

Comment:
I literally copied the contents of `lms/templates/navigation.html` in order to create this file.